manifests/working-template.yml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: Template
+metadata:
+  name: working
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+  labels:
+    miq.github.io/kubevirt-is-vm-template: true
+    miq.github.io/kubevirt-os: rhel-7
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: ${NAME}
+  spec:
+    template:
+      spec:
+        domain:
+          cpu:
+            cores: ${CPU_CORES}
+          resources:
+            requests:
+              memory: ${MEMORY}
+          machine:
+            type: q35
+          devices:
+            disks:
+            - name: registrydisk
+              volumeName: registryvolume
+              disk:
+                dev: vda
+            - name: cloudinitdisk
+              volumeName: cloudinitvolume
+              disk:
+                dev: vdb
+        volumes:
+          - name: registryvolume
+            registryDisk:
+              image: kubevirt/fedora-cloud-registry-disk-demo:latest
+          - name: cloudinitvolume
+            cloudInitNoCloud:
+              userDataBase64: I2Nsb3VkLWNvbmZpZwpwYXNzd29yZDogYXRvbWljCnNzaF9wd2F1dGg6IFRydWUKY2hwYXNzd2Q6IHsgZXhwaXJlOiBGYWxzZSB9Cg==
+parameters:
+- description: Name for the new VM
```

Comment:
Please reorder this list so that name/description are always in the same order :)

```
- name:
  description:
- name:
  description:
…
```

Reply:
OK